### PR TITLE
Added section about integrating collaboration plugins in React apps.

### LINKED ISSUE #5809: Add section about integrating collaboration plugins in React apps
## 📝 Provide a description of requested docs changes

*What is the purpose and what should be changed?*

This section is currently missing in React docs.

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Added section about integrating collaboration plugins in React apps. Closes #5809.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

File: docs/builds/guides/frameworks/react.md [hunk 1]
```diff
@@ -139,6 +139,20 @@ class App extends Component {
 export default App;
 ```
 
+### Note: Using the editor with collaboration plugins
```
Comment (AnnaTomanek) [inline docs/builds/guides/frameworks/react.md]:
```suggestion
### Using the editor with collaboration plugins
```

File: docs/builds/guides/frameworks/react.md [hunk 2]
```diff
@@ -139,6 +139,20 @@ class App extends Component {
...
+
+For such scenario we provide a few ready-to-use integrations featuring collaborative editing in React applications:
+
+- [CKEditor 5 with real-time collaboration features](https://github.com/ckeditor/ckeditor5-collaboration-samples/tree/master/real-time-collaboration-for-react)
+
+- [CKEditor 5 with the track changes feature](https://github.com/ckeditor/ckeditor5-collaboration-samples/tree/master/track-changes-for-react)
+
+It is not mandatory to build applications on top of the above samples, however, they should help you get started.
+
+Note: These integrations are meant to be as simple as possible, thus they do not use Create React App CLI, though you should have no problem starting from `CRA` after reading the below sections.
```
Comment (AnnaTomanek) [inline docs/builds/guides/frameworks/react.md]:
```suggestion
Note: These integrations are meant to be as simple as possible, so they do not use Create React App CLI. However, you should have no problem starting from `CRA` after reading the sections below.
```

File: docs/builds/guides/frameworks/react.md (hunk 1 above)
Comment (ma2ciek) [reply docs/builds/guides/frameworks/react.md]: 🤔 I agree that `Note:` should not be a part of the heading. But the two headings above contain the `Note:` part as well 😅

Comment (AnnaTomanek) [reply docs/builds/guides/frameworks/react.md]: Good point, let's remove the other two, then? ;)

Comment (ma2ciek) [reply docs/builds/guides/frameworks/react.md]: I've removed them. Please, take a look now :)

**Comment by AnnaTomanek** [comment]
Great job, thanks a lot! 🎉

**Pull request merged**